api/src/main/java/org/openmrs/util/DatabaseUpdater.java

```diff
@@ -68,7 +68,7 @@
 	
 	private static final Log log = LogFactory.getLog(DatabaseUpdater.class);
 	
-	private static final String CHANGE_LOG_FILE = "liquibase-update-to-latest.xml";
+	private static final String CHANGE_LOG_FILE = "liquibase-master.xml";
```

Comment:
Since this is really still an update file, it would nice for it to contain the word update,
how about `liquibase-update-to-latest-master.xml`?